src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -785,6 +784,35 @@ uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
     return UCS_OK;
 }
 
+static void
+uct_ib_mlx5_devx_query_port_select(uct_ib_mlx5_md_t *md,
+                                   uct_ib_port_select_mode_t *port_select_mode)
+{
+    char out[UCT_IB_MLX5DV_ST_SZ_BYTES(query_lag_out)] = {};
+    char in[UCT_IB_MLX5DV_ST_SZ_BYTES(query_lag_in)]  = {};
+    void *lag;
+    uint8_t lag_port_select_mode;
+    ucs_status_t status;
+
+    lag = UCT_IB_MLX5DV_ADDR_OF(query_lag_out, out, lag_context);
+    UCT_IB_MLX5DV_SET(query_lag_in, in, opcode, UCT_IB_MLX5_CMD_OP_QUERY_LAG);
+    status = uct_ib_mlx5_devx_general_cmd(md->super.dev.ibv_context, in,
+                                          sizeof(in), out, sizeof(out),
+                                          "QUERY_LAG", 0);
+    if (status != UCS_OK) {
+        return;
+    }
+
+    lag_port_select_mode
+           = UCT_IB_MLX5DV_GET(lag_context, lag, port_select_mode);
+
+    if (lag_port_select_mode <= UCT_IB_MLX5_LAG_MULTI_PORT_ESW) {
+        *port_select_mode = (uct_ib_port_select_mode_t)lag_port_select_mode;
```

Comment:
can we remove `(uct_ib_port_select_mode_t)` ?

Reply:
Done